### PR TITLE
Minor fix and improvement to preprocessor directives page.

### PR DESCRIPTION
**Pull request by davide-pi**
## Summary

Fix missing dash in an example of the region form of `#line` directive, that cause a syntax error [CS1003](https://msdn.microsoft.com/query/roslyn.query?appId=roslyn&k=k(CS1003))

image

Add aslo minor improvements on description of project level nullability.

> No related issues found

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/preprocessor-directives.md](https://github.com/dotnet/docs/blob/379630f3bb60297f1a920e70ac8ac2d310156b18/docs/csharp/language-reference/preprocessor-directives.md) | [C# preprocessor directives](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives?branch=pr-en-us-39876) |

### CODE REVIEW

File: docs/csharp/language-reference/preprocessor-directives.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Although the compiler doesn't have a separate preprocessor, the directives descr
 
 The `#nullable` preprocessor directive sets the *nullable annotation context* and *nullable warning context*. This directive controls whether nullable annotations have effect, and whether nullability warnings are given. Each context is either *disabled* or *enabled*.
 
-Both contexts can be specified at the project level (outside of C# source code). The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
+Both contexts can be specified at the project level (outside of C# source code) adding the `Nullable ` element to the `PropertyGroup` element. The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
```
Comment (BillWagner) [inline docs/csharp/language-reference/preprocessor-directives.md:50]: Let's link to the doc on `Nullable`:

```suggestion
Both contexts can be specified at the project level (outside of C# source code) adding the [`Nullable `](../compiler-options/language.md#nullable) element to the `PropertyGroup` element. The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
```

**Review by BillWagner — APPROVED** [review]
This looks great @davide-pi 

I added one link to the note on `Nullable`, and then I'll :shipit:

File: docs/csharp/language-reference/preprocessor-directives.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Although the compiler doesn't have a separate preprocessor, the directives descr
 
 The `#nullable` preprocessor directive sets the *nullable annotation context* and *nullable warning context*. This directive controls whether nullable annotations have effect, and whether nullability warnings are given. Each context is either *disabled* or *enabled*.
 
-Both contexts can be specified at the project level (outside of C# source code). The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
+Both contexts can be specified at the project level (outside of C# source code) adding the [`Nullable `](../compiler-options/language.md#nullable) element to the `PropertyGroup` element. The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
```
Comment (BillWagner) [inline docs/csharp/language-reference/preprocessor-directives.md:50]: wrong link:

```suggestion
Both contexts can be specified at the project level (outside of C# source code) adding the [`Nullable`](./compiler-options/language.md#nullable) element to the `PropertyGroup` element. The `#nullable` directive controls the annotation and warning contexts and takes precedence over the project-level settings. A directive sets the context(s) it controls until another directive overrides it, or until the end of the source file.
```

**Comment by davide-pi** [comment]
Thanks @BillWagner, for adding the link, and for the approval 💪

**Pull request merged**